dubbo-admin-backend/src/main/resources/application.properties

```diff
@@ -16,6 +16,8 @@
 #
 
 dubbo.registry.group=dubbo
+dubbo.configcenter.username=username
+dubbo.configcenter.password=password
```

Comment:
please change the style of configcenter to config-center, thanks

Reply:
Hi, I update this properties. But when I push it and run CI, I get an error '`The Travis CI build failed`'. and details is '`Failed to retrieve POM for org.apache.maven.surefire:surefire-junit4:jar:2.21.0: Could not transfer artifact org.apache.maven.surefire:surefire-junit4:pom:2.21.0 from/to central (https://repo.maven.apache.org/maven2): Connection reset`'. I do not change any POM file. Can you help me for this ? Thank you.